src/lib/models/ConfigModel.ts

```diff
@@ -83,50 +85,64 @@ abstract class YamlFormatter {
 		return yamlCode.replace(/(\r?\n|$)/, ` ${comment}$1`);
 	}
 
-	private toYamlField(fieldName: string, userValue: unknown, defaultValue: unknown): string {
-		if (looksLikeAPathValue(userValue) && userValue === defaultValue) {
-			// We special handle a path value when it is equal to the default value, making it equal null because
-			// chances are it is a derived file or folder value based on the specific environment that we do not want to
-			// actually want to hard code since checking in the config to CI/CD system may create a different value
-			const commentText: string = getMessage(BundleName.ConfigModel, 'template.last-calculated-as', [JSON.stringify(userValue)]);
-			return this.toYamlUncheckedFieldWithInlineComment(fieldName, null, commentText);
-		} else if (JSON.stringify(userValue) === JSON.stringify(defaultValue)) {
-			return this.toYamlUncheckedField(fieldName, userValue);
+	private toYamlFieldUsingFieldDescription(fieldName: string, resolvedValue: unknown, fieldDescription: ConfigFieldDescription): string {
```

Comment:
Using the name "resolvedValue" instead of "userValue" because it is post-resolution of the config... which mean it could be a calculated value like our python_command value for example.